### PR TITLE
Move REST endpoint to v1

### PR DESCRIPTION
**Pull request by fmarco76**
The current REST implemenation is associated to /v1 path and the /rest path is a redirect to the /v1.

This would make easier to switch between REST implementations.

### CODE REVIEW

File: base/kra/shared/webapps/kra/WEB-INF/web.xml [hunk 1]
```diff
@@ -32,7 +34,7 @@
     <security-constraint>
         <web-resource-collection>
             <web-resource-name>Agent Services</web-resource-name>
-            <url-pattern>/rest/agent/*</url-pattern>
+            <url-pattern>/v1/agent/*</url-pattern>
```
Comment (edewata) [inline base/kra/shared/webapps/kra/WEB-INF/web.xml:37]: Do we need to keep the old URL?

**Review by edewata — APPROVED** [review]
I have one comment, but it LGTM.

File: base/kra/shared/webapps/kra/WEB-INF/web.xml (hunk 1 above)
Comment (fmarco76) [reply base/kra/shared/webapps/kra/WEB-INF/web.xml:35]: > Do we need to keep the old URL?

You are right, it is not needed. Removed!

**Comment by fmarco76** [comment]
@edewata Thanks! I have updated the path for the classes.
The sonar test is failing from some code in v1 REST APIs I have not modified but moving the files they get re-evaluated.
I am ignoring it in this PR.

**Pull request merged**